distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -2909,11 +2909,8 @@ Recommendations for Framework Development
 ===== Console Commands
 
 ${branding} supports development of custom console commands.
-For more information, see the Karaf website on http://karaf.apache.org/manual/latest-2.2.x/developers-guide/extending-console.html[Extending the Console]. 
 
-===== Custom ${branding} Console Commands
-
-${branding} includes custom commands for working with the Catalog, as described in the Console Commands section.
+${branding} includes custom commands for working with the Catalog, as described in the <<Console Commands>> section.
```

Comment:
Yeah, I'll need to fix it. It works correctly in the complete documentation, but it doesn't work in the standalone extending.html, because it doesn't include the managing section.